open_source_policy.md

```diff
@@ -36,7 +36,10 @@ Sensitive reference information can take several forms. A non-exhaustive list of
 1. Information which identifies specific accounts or resources (AWS Account numbers including within ARNs, S3 bucket names, etc). For the avoidance of doubt, it is fine to substitute obviously-made-up dummy values for the real values where necessary.
 1. Information marked as OFFICIAL-SENSITIVE or higher, as per the [Government Security Classifications](https://www.gov.uk/government/publications/government-security-classifications).
 1. IP Addresses (note that DWP considers these to be OFFICIAL-SENSITIVE).
-1. Service information which could enable an attacker to identify service vulnerabilities (software versions and patch levels, etc).
+1. Service information which could enable an attacker to identify service vulnerabilities (software versions and patch levels, etc). However, often exact versions of packages and dependencies are required. These can be permitted when other controls are in place. The following can be used to mitigate this risk:
+    1. Encrypt files containing specific versions (e.g. `package-lock.json`) using [git-secret](https://git-secret.io)
```

Comment:
This beer is not ready yet! We're not done reviewing git-secret. That's a decent suggestion, but we need to consider how useful it is in the context of a (partially) open-source world where much of the code and some of the artefacts generated are, rightly, stored in public repositories. This makes it easy for an attacker to scan repos and artefacts for vulnerabilities and simply highlights the need for proper defence-in-depth for design and reactive vulnerability management; we can no longer hide behind the illusion of security which we get by keeping a closed shop. That said, there will be real value in encrypting some of this information for some cases as you suggest. Perhaps a quick chat about this one.